### ISSUE TITLE
Process and load data

### ISSUE THREAD

**Issue by TamaraAtanasoska** [TamaraAtanasoska/SemEval-2022-Task-8-Multilingual-News-Article-Similarity#4]
The `.htm`l and `.json` files of the data are downloaded and stored in folders grouped by their last digits. The pair information can be taken from the `.csv` files with the links provided by the competition and currently in the data folder, including the similarity rating/gold standard label for the train and eval data respectively. 

Tasks around data: 

- find what parts of the data are relevant for the models or for the preprocessing 
- exclude pairs that have missing articles 
- load data into the dataloader 

This is a a minimum viable data usage, according to some of the papers there could be a few more steps before the data is packed in the Dataloader.
